ls1intum/Artemis
# `Plagiarism cases`: Send emails for notifications posts in plagiarism cases

### Checklist
#### General


#### Server

### Motivation and Context


Currently students are not notified via email that they received a plagiarism case, even though they should. 

### Description

This PR adds email templates and makes sure that they’re sent when a plagiarism case is notified. 

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Student
- Your own account
- 1 Exercise

1. Log in to Artemis
2. Participate with both students (your account and other student)
3. Check for plagiarism in that exercise (you might have to decrease the needed similarity)
4. Confirm the case for your user
5. Notify your user account about the case in the plagiarism overview of the course
6. Set a verdict
7. Check the emails in your mail client

### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage


| Class/File | Branch | Line |
|------------|-------:|-----:|
| SingleUserNotificationService.java | 93% | 97% |
